modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
+//#TINY-9510
+
+In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written `regexp` rule in the typograf library for the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.
+
+To fix this, {productname}, updated the `regexp` in the typograf library. As a result, spaces between commas and quotes no longer appear.
```

Comment:
```suggestion
=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
//#TINY-9510

In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written Regular Expression rule in the https://github.com/typograf[Typograf] library used by the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.

To fix this, {productname} updated the Regular Expression rule in the Typograf library. As a result, spaces between commas and quotes no longer appear.
```
Copy-edits.

Also, this isn’t a TinyMCE fix. It is a fix to the [Typography plugin](https://github.com/tinymce/typography/blob/main/CHANGELOG.md).

It goes in the _Accompanying Premium Plugin changes_ section, in an _Advanced Typography_ sub-section.

Reply:
@brian-forte Good pickup, ill move the content to the required section